### ISSUE TITLE
investigate lag in scrolling with mouse vs scrolling with scroll bars

### ISSUE THREAD

**Issue by russorat** [influxdata/chronograf#4247]
as a logviewer user
with viewer permissions or higher
when i scroll using my mouse
then i would like the same behavior as if i was scrolling via the scrollbar

for some reason, scrolling with the mouse (or two fingers on the mac trackpad) is slow and choppy when loading log lines. When i click on the scrollbars, and drag, the scrolling is much smoother. I'd like the same smooth experience when scrolling with the mouse or trackpad.

**Comment by jaredscheib** [comment]
In spiking on this with @AlirieGray and @OfTheDelmer for an hour, it wasn't clear whether this could be resolved, or how to resolve it if it is possible. I tried changing the `transform` to see if hardware acceleration could help, a la https://github.com/bvaughn/react-virtualized/issues/453#issuecomment-288687484, that didn't help. We found a bunch of articles about ScrollSync and JavaScript being informed async vs the browser event triggering synchronous responses, but this doesn't seem to be the problem. We'd welcome any further ideas here.
